### ISSUE TITLE
Warn user when he has installed `remark` instead of `remark-cli`

### ISSUE THREAD

**Issue by RaitoBezarius** [coala-analyzer/coala-bears#586]
When using the Markdown Bear, you'd be tempted to run a `npm install -g remark` as the tool is saying that `remark` is not installed.

Though, the exact package is `remark-cli`, the fact is the bear is wrongly used the fake `remark` tool which will just display a "No you should uninstall this and install `remark`", causing coala to remove *everything* in your Markdown file if you decide to apply patches.

Not super nice UX !

**Comment by sils1297** [comment]
Indeed, we probably wanna add a simple note to the message and be done with it? Like be sure to install the remark-cli package and not remark.

@adrianzatraneu can we already show the generated command?

**Comment by sils1297** [comment]
CC @Adrianzatreanu can we have the failure messages generated from the requirements attribute?

**Comment by jayvdb** [comment]
This sounds like a specialised Requirement class for dependency_management.
